src/ResourceManagement/Billing/Billing.Tests/ScenarioTests/InvoicesTests.cs

```diff
@@ -0,0 +1,100 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Billing.Tests.Helpers;
+using Microsoft.Azure.Management.Billing;
+using Microsoft.Azure.Management.Billing.Models;
+using Microsoft.Azure.Management.Resources;
+using Microsoft.Rest.ClientRuntime.Azure.TestFramework;
+using System;
+using System.Linq;
+using System.Net;
+using Xunit;
+
+namespace Billing.Tests.ScenarioTests
+{
+    public class InvoicesTests : TestBase
+    {
+        const string DownloadUrlExpand = "downloadUrl";
+        const string RangeFilter = "invoicePeriodEndDate ge 2017-01-31 and invoicePeriodEndDate le 2017-02-28";
```

Comment:
yes, the filter is valid in SDK. For PS, we decided to have different behavior for the 1st release. If customer gives feedback on requesting date range filter, we will add it in later release. The reason we dont want to add it in 1st release is because we think it's a bit confusing and since we only have very few data (3 month data) for now, the data range filter might not be very useful.